app/views/eventDetails/paymentOptions.html

```diff
@@ -5,6 +5,15 @@ <h2 class="page-title border" translate>Payment Options</h2>
     <h3 class="panel-title" translate>Credit Card</h3>
   </div>
   <div class="panel-body">
+    <div class="form-group">
+      <div class="col-sm-12 stacked-spacing-col-sm" ng-if="conference.paymentGatewayType === 'AUTHORIZE_NET'">
+        <label translate>Select A Payment Gateway Type:</label>
+        <select class="form-control" ng-model="conference.paymentGatewayType">
```

Comment:
These options could be generated by the data in `$scope.paymentGateways`. I won't say that you have to, but it would reduce duplication of the labels. You would use `ng-options` to specify an array of options generated from `$scope.paymentGateways`. There would be one option per entry, with the key being the option value and the `name` field being option label. Let me know if you decide to go that route and need help figuring it out.